osu.Game/Screens/Menu/OsuLogo.cs

```diff
@@ -144,7 +144,6 @@ public OsuLogo()
                                                     RelativeSizeAxes = Axes.Both,
                                                     Origin = Anchor.Centre,
                                                     Anchor = Anchor.Centre,
-                                                    Alpha = 0.5f,
```

Comment:
what's the deal with these? were they never applying due to missing draw node logic?

Reply:
They were applying correctly, but is less obvious when used with brighter colours. 

When using `#ff0064` (a rather dark colour), this applied on top of the 0.2 opacity of the actual bars, the final result was way too transparent. I chose to just remove it, since the relative difference when using brighter colours is negligible.

Reply:
that doesn't make sense though. going to require further investigation.

Reply:
The reasoning I had was to make lower luminosity colours a bit more visible with this change, and it's pretty much just that. But after a bit of staring, I'm kinda feeling that the blame should instead lie with the colors used, because trying to accommodate every single colour would be endless.

Reply:
I still question the merits of this being an issue, unless there are plans to expose an alpha value to users in the config but as it currently stands users have access to the top end of that additive sprite and adding more alpha robs users of that especially if they're using a custom texture. If they needed more transparency they could have lowered the value like in the original issue to something like 127 across the board (black = transparency). That background also wasn't doing them any favors adding white against a light blue sky so the fix here might not even help. Stable has the same moire issue and alpha just hides the problem but it's not what I consider a fix, whole visualizer probably needs to be built in a buffer before an aa pass to properly get rid of it but that's beyond my skillset and probably outside the current scope of things.